tests/pytests/functional/cli/test_salt.py

```diff
@@ -17,6 +17,7 @@ def test_versions_report(salt_cli):
                 section[key] = section[key].strip()
 
     ret = salt_cli.run("--versions-report")
+    assert ret.returncode == 0
```

Comment:
Can we test "relenv" exists in the output of `--versions-report` if we are testing a onedir build?

Reply:
Oh, specifically of we're on a Onedir right?

Reply:
Pushed a test update.